terraform/deployments/govuk-publishing-infrastructure/db_backup_iam.tf

```
@@ -0,0 +1,54 @@
+locals {
+  db_backup_service_account_name = "db-backup"
+  env_also_reads_from = {
+    "production"  = "govuk-production-database-backups-replica"
+    "staging"     = "govuk-production-database-backups"
+    "integration" = "govuk-staging-database-backups"
```

optional/trivial: can drop some unnecessary quotes here (dunno why tf fmt doesn't deal with this automatically 🤷)
```suggestion
    production  = "govuk-production-database-backups-replica"
    staging     = "govuk-production-database-backups"
    integration = "govuk-staging-database-backups"
```